ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,125 @@
+                                    sling:resourceType="fd/af/authoring/components/granite/form/formsmultifield"
+                                    fieldDescription="Enter Save Value - Enums"
+                                    fieldLabel="Save Values - Enums">
+                                <field
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                        name="./enum"
+                                        required="{Boolean}false"/>
+                            </enum>
+                            <enumNames
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:orderBefore="placeholder"
+                                    sling:resourceType="fd/af/authoring/components/granite/form/formsmultifield"
+                                    fieldDescription="Enter Display Values (EnumNames) corresponding to their respective Save values"
+                                    fieldLabel="Display Values - EnumNames">
+                                <field
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                        name="./enumNames"
+                                        required="{Boolean}false"/>
+                            </enumNames>
+                            <multiSelect
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:orderBefore="visible"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                    name="./multiSelect"
+                                    text="Allows multiple selection"
+                                    value="true"/>
+                            <defaultValue
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:orderBefore="visible"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                    fieldDescription="Must be one of the options."
+                                    fieldLabel="Default Value"
+                                    name="./default"/>
+                            <multiValuePath
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:orderBefore="visible"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                    fieldLabel="Items Load Path"
+                                    name="./optionsLoadPath"/>
```

Comment:
missing in the docs

Reply:
Okay I will remove this from the dialog.